frontend/actions/currentSection/currentSection_action.jsx

```diff
@@ -0,0 +1,6 @@
+export function sectionSelector(section){
```

Comment:
So this would look like `frontend/actions/current_section/current_section_action.jsx`